flycheck.el

```diff
@@ -7722,8 +7722,8 @@ See URL `https://github.com/commercialhaskell/stack'."
             source)
   :error-patterns
   ((warning line-start (file-name) ":" line ":" column ":"
-            (or " " "\n    ") "Warning:"
-            (optional " " "[" (id (one-or-more not-newline)) "]")
+            (or " " "\n    ") (in "Ww") "arning:"
+            (optional " " "[" (id (one-or-more not-newline)) "] ?")
```

Comment:
The ws change is just wrong, `flycheck-compile` is just displaying it wrong, maybe because it underlines where the carriage return is. On the cmdline, there's no extra space when I copypaste the `flycheck-compile` call. I think I was flailing when first trying to get this to work, frankly; and the bad "?" syntax proves that the change was a) broken but b) harmless. Buh-bye :)